main/solution/backend/config/infra/cloudformation.yml

```diff
@@ -801,9 +801,20 @@ Resources:
                 - ec2:ModifyImageAttribute
                 - ec2:DescribeImages
                 - ec2:DescribeInstances
+                - ec2:DescribeSecurityGroups
+                - ec2:RevokeSecurityGroupIngress
+                - ec2:AuthorizeSecurityGroupIngress
```

Comment:
Do we need to update our documentation to call out that member account roles need to add these permissions?

Reply:
I'll be updating this in the ChangeLog doc.